options/options.c

```diff
@@ -54,8 +54,8 @@
 #include "video/out/drm_common.h"
 #endif
 
-#if HAVE_GL
-#include "video/out/opengl/hwdec.h"
+#if HAVE_GPU
+#include "video/out/gpu/hwdec.h"
 #endif
```

Comment:
Hm, probably time to make HAVE_GPU required and to remove the ifdefs.

Reply:
As long as you're okay with a swarm of angry emails from debian 5 users who had the perfect workflow with -vo=drm and don't want to install some shitty graphical protocol just so they can use mpv in the terminal.

Reply:
They can still disable both GL and Vulkan. Just not the generic code. Anyway, still something for a separate commit, I guess.